src/subscribe-form.hbs

```diff
@@ -1,4 +1,7 @@
 <form id="{{id}}" data-subscribe-instance="{{instanceId}}" class="{{namespace}}">
+{{#each hiddenFields}}
+<input class="{{../namespace}}__input {{../namespace}}__input--hidden" type="hidden" name="{{name}}" value="{{value}}">
```

Comment:
Well, in theory those classes could be used for JS too. I was using them as handles for the tests, though that's not really necessary. They're not really necessary here, just added for consistency with the rest of the elements.